gematria/granite/graph_builder_model_inference.cc

```diff
@@ -128,6 +129,7 @@ constexpr std::string_view GetInputTensorName(const InputTensor tensor) {
       return name;
     }
   }
+  return kUnknownTensorName;
```

Comment:
`kPossibleInputTensors` should be exhaustive, so this should never happen. I think we should have an assertion or something similar here.

Either `assert(false && "Unexpected InputTensor value");` or some way to invoke `__builtin_unreachable()`, e.g. `LLVM_BUILTIN_UNREACHABLE`.

Reply:
I've switched to just using `__builtin_unreachable` directly as I think any of the compilers we will use here should have support for it.